### PR TITLE
Fix #8095: Misleading command descriptions

### LINKED ISSUE #8095: Misleading command descriptions
The descriptions of what these command do is very misleading. The Powershell equivalent docs are much more helpful - please align them to be more consistent.

For example:

Powershell cmdlet `Remove-AzKeyVaultManagedStorageAccount`:
https://docs.microsoft.com/en-gb/powershell/module/az.keyvault/remove-azkeyvaultmanagedstorageaccount?view=azps-1.0.0

says:

> Removes a Key Vault managed Azure Storage Account and all associated SAS definitions.

CLI equivalent:
https://docs.microsoft.com/en-us/cli/azure/keyvault/storage?view=azure-cli-latest#az-keyvault-storage-remove

says:

> Deletes a storage account.

To my mind, this implies that the Storage Account resource will be deleted which is definitely not what happens!

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: a2d724c9-6606-2a86-b8e8-43c01b1a42e1
* Version Independent ID: 22c38bcb-78d6-9347-00e7-86f56b2674df
* Content: [az keyvault storage](https://docs.microsoft.com/en-us/cli/azure/keyvault/storage?view=azure-cli-latest#az-keyvault-storage-remove)
* Content Source: [src/command_modules/azure-cli-keyvault/azure/cli/command_modules/keyvault/_help.py](https://github.com/Azure/azure-cli/blob/dev/src/command_modules/azure-cli-keyvault/azure/cli/command_modules/keyvault/_help.py)
* Service: **key-vault**
* GitHub Login: @rloutlaw
* Microsoft Alias: **routlaw**

### PR DESCRIPTION
**Pull request by bim-msft**
---
Fix #8095 

This checklist is used to make sure that common guidelines for a pull request are followed.

- [x] The PR has modified HISTORY.rst describing any customer-facing, functional changes. Note that this does not include changes only to help content. (see [Modifying change log](https://github.com/Azure/azure-cli/tree/master/doc/authoring_command_modules#modify-change-log)).

- [x] I adhere to the [Command Guidelines](https://github.com/Azure/azure-cli/blob/dev/doc/command_guidelines.md).

### CODE REVIEW

File: src/azure-cli/azure/cli/command_modules/keyvault/_help.py [hunk 1]
```diff
@@ -193,6 +193,11 @@
         --auto-regenerate-key --regeneration-period P90D  --resource-id $id
 """
 
+helps['keyvault storage remove'] = """
+type: command
+short-summary: Remove a Key Vault managed Azure Storage Account and all associated SAS definitions.
```
Comment (qwordy) [inline src/azure-cli/azure/cli/command_modules/keyvault/_help.py]: Storage Account -> storage account

Comment (bim-msft) [reply src/azure-cli/azure/cli/command_modules/keyvault/_help.py]: @qwordy I copied this from PowerShell docs for consistency: https://docs.microsoft.com/en-gb/powershell/module/az.keyvault/remove-azkeyvaultmanagedstorageaccount?view=azps-3.1.0&viewFallbackFrom=azps-1.0.0

Comment (qwordy) [reply src/azure-cli/azure/cli/command_modules/keyvault/_help.py]: It's not a big problem.
This is the first result in search engine.
https://docs.microsoft.com/en-us/azure/storage/common/storage-account-overview

**Pull request merged**